src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/ExportTiles/ExportTiles.xaml.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright 2017 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.UI;
+using System;
+using System.Linq;
+using System.Windows;
+using System.Windows.Media;
+
+namespace ArcGISRuntime.WPF.Samples.ExportTiles
+{
+    public partial class ExportTiles
+    {
+        // URL to the service tiles will be exported from
+        private Uri _serviceUri = new Uri("https://sampleserver6.arcgisonline.com/arcgis/rest/services/World_Street_Map/MapServer");
+
+        public ExportTiles()
+        {
+            InitializeComponent();
+
+            // Call a function to set up the map
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create the tile layer
+            ArcGISTiledLayer layer = new ArcGISTiledLayer(_serviceUri);
+
+            // Load the layer
+            await layer.LoadAsync();
+
+            // Create the basemap with the layer
+            Map myMap = new Map(new Basemap(layer));
+
+            // Assign the map to the mapview
+            MyMapView.Map = myMap;
+
+            // Create a new symbol for the extent graphic
```

Comment:
Give the user a little more comments as what is really happening here with this graphic. 
This graphic is the red box that appears on the MapView and will be used to define the extent of tiles that are retrieved from the ExportTileCacheJob.